src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -53,6 +53,7 @@
         ("vit_mae", "ViTFeatureExtractor"),
         ("segformer", "SegformerFeatureExtractor"),
         ("convnext", "ConvNextFeatureExtractor"),
+        ("van", "ConvNextFeatureExtractor"),
```

Comment:
Re-using this for all models? ;)

You're sure it expects the same transforms at evaluation time?

Reply:
it was trained on imagenet